versioned_docs/version-0.5.0/release-notes/release-notes-for-kubeslice-oss-0.5.1.mdx

```diff
@@ -0,0 +1,19 @@
+# Release Notes for KubeSlice OSS 0.5.0 v1
+
+*Release date: 06 Jan 2023*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New
+These release notes describe the new changes and enhancements in this version.
+
+### Latest Supported Kubernetes Version 
+The latest Kubernetes version that we support from this release is version [1.24](https://v1-24.docs.kubernetes.io/). 
+
+### Enhancements 
+* The **kubeslice-cli** tool now allows you to uninstall Kubeslice components using a custom topology file.
```

Comment:
now allows you to => now allows uninstalling

Reply:
updated